jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -1,8 +1,5 @@
 pipeline {
     agent none
-    environment {
-        OPENSEARCH_DASHBOARDS_BUILD_ID = "${BUILD_NUMBER}"
-    }
     triggers {
       parameterizedCron '''
         H */2 * * * %INPUT_MANIFEST=1.1.0/opensearch-dashboards-1.1.0.yml
```

Comment:
So dashboards still build 1.1.0 bundle?

Reply:
It's the first time we build dashboards and we want to be testing against stable code. Once we have verified that our process yields the same output as the build we had assembled manually, we can stop doing it here and move to 1.2.